src/components/popupWithConfirmation.js

```diff
@@ -1,42 +1,38 @@
 import Popup from "./Popup.js";
 
-export default class popupWithConfirmation extends Popup {
-  constructor(popupSelector) {
+export default class PopupWithConfirmation extends Popup {
+  constructor(popupSelector, loadingText) {
     super(popupSelector);
     this._popupForm = this._popup.querySelector(".modal__container");
     this._confirmButton = this._popupForm.querySelector(
       ".modal__confirm-button"
     );
     this._confirmButtonText = this._confirmButton.textContent;
+    this._loadingText = loadingText;
   }
 
   close() {
     super.close();
```

Comment:
You shouldn’t create the **same methods** of the parent class if you don’t change it inside and only call it with `super`. The parent methods are available in any inheritor of the class.